__tests__/app.test.js

```diff
@@ -60,3 +60,24 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
```

Comment:
There should be a body property on the object, please add an assertion for it in this test